cmssw-vectorization.file

```diff
@@ -1,2 +1,2 @@
-%define vectorized_packages zlib fastjet tensorflow-sources tensorflow OpenBLAS rivet gbl
+%define vectorized_packages zlib fastjet tensorflow-sources tensorflow tensorflow-xla-runtime OpenBLAS rivet gbl
```

Comment:
@riga, please also update https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_14_1_X/master/vectorization/cmsdist_packages.py#L14-L22 and include ` tensorflow-xla-runtime`  in the list

Reply:
Added :+1:

Reply:
ah sorry, I forgot to mention, please also add `scram-tools.file/tools/tensorflow-xla-runtime/vectorized.tmpl`  file same as https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_14_1_X/master/scram-tools.file/tools/tensorflow/vectorized.tmpl except that the tool `name` should be `tensorflow-xla-runtime_@TOOL_VECTORIZATION@`